docs/visualization/visualize_unit_metrics.ipynb

```diff
@@ -122,10 +122,10 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    # converts 'electrodes' object from NWB file into pandas DataFrame (allows for easier analysis of data)
</pre><p>probably not necessary, but maybe could be added as a sentence in the markdown.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>